packages/shared/services/types.ts

```diff
@@ -16,18 +16,12 @@ limitations under the License.
 
 export type AuthProviderType = 'oidc' | 'saml' | 'github';
 
-export type Auth2faType =
-  | 'u2f'
-  | 'otp'
-  | 'off'
-  | 'optional'
-  | 'on'
-  | 'webauthn';
+export type Auth2faType = 'otp' | 'off' | 'optional' | 'on' | 'webauthn';
 
 // PreferredMfaType is used to determine which MFA option
 // is preferred when more than one option can be available
 // and only one should be preferred.
-export type PreferredMfaType = 'webauthn' | 'u2f' | '';
+export type PreferredMfaType = 'webauthn' | '';
```

Comment:
Is this needed now that there's only one type? Or are there two types, one of which is represented by `''`? Seems like it should either be able to be removed, or at least a comment should clarify what `''` means.

Reply:
i added a TODO to remove it later, it will be in next PR b/c there are several places it's affected (it [doesn't do anything](https://github.com/gravitational/webapps/pull/711/commits/f7b1a313530a10524af5268ee2d80896617ad75d#diff-3120151bac13eea22702b220d01a4b2f88201a6ca9f4114ce0d2953986b2e72dR51) atm)